internal_transfers/actions/src/dune.ts

```diff
@@ -0,0 +1,38 @@
+import { QueryParameter, DuneClient } from "@cowprotocol/ts-dune-client";
+import { MinimalTxData } from "./accounting";
+
+const TX_DATA_QUERY_ID = 2492342;
+
+export async function getSampleSet(
+  dune: DuneClient,
+  // YYYY-MM-DD
+  dateFrom: string,
+  dateTo: string
+): Promise<MinimalTxData[]> {
+  const historicalTransactionData = await dune
+    .refresh(TX_DATA_QUERY_ID, [
+      QueryParameter.date("DateFrom", `${dateFrom} 00:00:00`),
+      QueryParameter.date("DateTo", `${dateTo} 00:00:00`),
+    ])
+    .then((executionResult) => executionResult.result);
+  if (!historicalTransactionData) {
+    throw new Error("Failure");
+  }
+
+  return historicalTransactionData!.rows.map((row: Record<string, any>) => {
```

Comment:
You also don't need `!` here as you throw an error when it's `falsy`

Reply:
oh ya... Would it also be ok to remove the error throwing and expect the ! to throw instead? Or is it better to handle the error as is already done?

Reply:
```suggestion
  return historicalTransactionData.rows.map((row: Record<string, any>) => {
```